modules/character_net_controller/character_net_controller.cpp

```diff
@@ -824,14 +838,13 @@ void PlayerController::physics_process(real_t p_delta) {
 	// internet connection we can't keep accumulates inputs up to infinite
 	// otherwise the server will difer too much from the client and we
```

Comment:
Not urgent to fix, but I found a typo:
```suggestion
	// otherwise the server will differ too much from the client and we
```

Reply:
Ty fixed